packages/repository/src/__tests__/unit/repositories/relations-helpers/resolve-has-many-through-metadata.unit.ts

```diff
@@ -13,22 +13,52 @@ import {
   RelationType,
 } from '../../../..';
 import {
-  createTargetConstraint,
-  createThroughConstraint,
-  createThroughFkConstraint,
+  createTargetConstraintFromThrough,
+  createThroughConstraintFromSource,
+  createThroughConstraintFromTarget,
+  getTargetKeysFromThroughModels,
   HasManyThroughResolvedDefinition,
   resolveHasManyThroughMetadata,
 } from '../../../../relations/has-many/has-many-through.helpers';
 
 describe('HasManyThroughHelpers', () => {
-  context('createThroughConstraint', () => {
+  context('createThroughConstraintFromSource', () => {
     it('can create constraint for searching through models', () => {
-      const resolved = resolvedMetadata as HasManyThroughResolvedDefinition;
-      const result = createThroughConstraint(resolved, 1);
+      const result = createThroughConstraintFromSource(relationMetaData, 1);
       expect(result).to.containEql({categoryId: 1});
     });
   });
-  context('createTargetConstraint', () => {
+  context('getTargetKeysFromThroughModels', () => {
+    it('can return the target fk value of a given through instance', () => {
```

Comment:
`can return the target fk value of a given through instance` -> `returns the target fk value of a given through instance`

Similarly, update wherever applicable.